clients/src/test/java/org/apache/kafka/clients/consumer/internals/FetcherTest.java

```diff
@@ -252,6 +230,11 @@ public void teardown() throws Exception {
         }
     }
 
+    private int sendFetches() {
+        offsetFetcher.validatePositionsOnMetadataChange();
```

Comment:
Got it. The tests are probably the main reason we called the method from `sendFetches`. What we're really depending on is the transition to `Fetching` in `SubscriptionState`.